build-tools/packages/build-cli/src/library/typescriptApi.ts

```diff
@@ -99,5 +117,73 @@ export function getApiExports(sourceFile: SourceFile): ExportRecords {
 		}
 	}
 
+	// If we have found levels for all things exported, then nothing else left to look for.
+	if (records.unknown.size === 0) {
+		return records;
+	}
+
+	// Otherwise, look for some special cases where level tagging may appear in source itself.
+
+	// Note that these are not exported declarations, but specifically
+	// given file's export declarations.
+	const exportDeclarations = sourceFile.getExportDeclarations();
+	for (const exportDeclaration of exportDeclarations) {
+		// console.log(`export @ ${exportDeclaration.getStartLineNumber()} has children:\n\t${exportDeclaration.getChildren().map((c) => c.getKindName()).join('\n\t')}`);
```

Comment:
Remove?

Reply:
Nah - it is good for debugging - I was going to comment it though :)